node/pkg/p2p/p2p.go

```diff
@@ -559,7 +565,7 @@ func Run(
 				continue
 			}
 
-			logger.Debug("received message",
+			logger.Info("received message",
```

Comment:
Did you purposefully change this log level? I'm assuming yes, but it is totally unrelated to this PR.

Reply:
Nope. Undid it.